### ISSUE TITLE
Trying to access the 'submit' button to add attribute aria-label for 508 compliance

### ISSUE THREAD

**Issue by r2t2** [OvidijusParsiunas/deep-chat#465]
I'm trying to add the attribute 'aria-label' to the 'submit' button for 508 compliance of our site.  I did see issue #385 and cannot get it to work for me.  I have been unable to access any element below the chat-view div.  FYI, this is an Angular project (typescript).

```
  let parent = document.querySelector("deep-chat").shadowRoot.childNodes[0];
  let nodes = parent.childNodes;
  console.log("parent: ", parent);
  console.log("nodes length: ", nodes.length);
  console.log("nodes: ", nodes);
```
nodes contains a NodeList[] with 1 node -- div#chat-view, but it also reports the length as 0 and any attempt to drill deeper results in undefine/null responses.  

Console responses from above:

<img width="304" height="123" alt="Image" src="https://github.com/user-attachments/assets/95998461-fe09-4731-8ce6-60ac701355a3" />

Also, direct access i.e. xxxxx.querySelector('.input-button') or ('div.input-button') or any other combination will not access the element. 

Do you have any ideas or direction on accessing the element?  

<img width="491" height="392" alt="Image" src="https://github.com/user-attachments/assets/b6190999-4952-4727-ad0f-74d98a658672" />

**Comment by r2t2** [comment]
Thank you for your time. 

Your response is pretty much what I have done. Here is the results from the "copy selector":

`#input > div.input-button-container.outer-button-container.text-input-container-right-adjustment > div`

and this is the results of testing:

When I ran this command:
 `let parent = document.querySelector("deep-chat").shadowRoot.querySelector("div");`
parent was populated with the element correctly.

If I go any deeper, i.e.
 `let parent = document.querySelector("deep-chat").shadowRoot.querySelector("div.input-button-container.outer-button-container.text-input-container-right-adjustment > div");`
or 
 `let parent = document.querySelector("deep-chat").shadowRoot.querySelector("#input > div.input-button-container.outer-button-container.text-input-container-right-adjustment > div");`

I get null returned.  I am still unable to get to the element through code.

Thanks again,
Roger

**Comment by r2t2** [comment]
Yes, that returns NULL too.

**Comment by OvidijusParsiunas** [comment]
I have tested this in an Angular project and `document.querySelector("deep-chat").shadowRoot.querySelector("#input > div:nth-child(5) > div")` seems to work fine for me. I am not sure if there is really much that I can do.
Is your website published anywhere?

**Comment by r2t2** [comment]
No, it didn't help. :(

<img width="701" height="471" alt="Image" src="https://github.com/user-attachments/assets/100fa7fc-2255-407e-942d-02e783998e46" />

And the code I tried:
```
    const subButton = document
      .querySelector('deep-chat').shadowRoot
      .querySelector('.input-button.input-button-svg.outside-right.submit-button-enlarged.disabled-button');
    console.log("subButton: ", subButton);
```

and the result is still `NULL`
